R/Google.R

```diff
@@ -84,6 +84,12 @@ Google <- R6::R6Class("Google",
       "total_recovered",
       "total_tested"
     ),
+    #' @field source_text Plain text description of the source of the data
+    source_text = "O. Wahltinez and others",
```

Comment:
if people are going to add to citation etc we should go with et al. here rather than others to fit in the usual format.